ntoskrnl/mm/ARM3/virtual.c

```diff
@@ -1348,7 +1348,12 @@ MmFlushVirtualMemory(IN PEPROCESS Process,
 
     UNIMPLEMENTED;
 
-    return STATUS_NOT_IMPLEMENTED;
+    // Report actual state.
```

Comment:
This is not enough alone, and might be somewhat superfluous.
Yet, it's "expected", more explicit and safer.
Note: Actual function code used to set `IoStatusBlock->Information`.